src/event-timer.ts

```diff
@@ -33,6 +33,24 @@ interface PageEventStatus {
 	commercialModulesLoaded: boolean;
 }
 
+type ConnectionType = "bluetooth" | "cellular" | "ethernet" | "mixed" | "none" | "other" | "unknown" | "wifi";
```

Comment:
I thought double quotes would get flagged by Prettier?

Reply:
That is surprising. It isn't flagging and CI passes.

Reply:
Strange, I now wonder if we ever actually check Prettier in CI, or just rely on people auto-formatting on save 🤔